examples/basics/plotting/plot.py

```diff
@@ -11,13 +11,13 @@
 
 fig = vp.Fig(size=(600, 500), show=False)
 
-# Plot the target square wave shape 
+# Plot the target square wave shape
```

Comment:
I get the left side of the ylabel text cut off, do you?

![screenshot from 2016-02-18 14-15-31](https://cloud.githubusercontent.com/assets/2365790/13155107/16a361e6-d64a-11e5-9f9a-3eac45c6aa3d.png)

Otherwise it works well